KnowYourArmorPatcher/Program.cs

```diff
@@ -246,13 +246,12 @@ List<string> GetRecognizedKeywords(Armor armor)
 
                 if (npc.Race.TryResolve(state.LinkCache, out var race) && race.EditorID != null && armorRaces.Contains(race.EditorID))
                 {
-                    var perk = perkLink.DeepCopy();
                     var npcCopy = state.PatchMod.Npcs.GetOrAddAsOverride(npc);
                     if (npcCopy.Perks == null) npcCopy.Perks = new ExtendedList<PerkPlacement>();
                     PerkPlacement p = new PerkPlacement
                     {
                         Rank = 1,
-                        Perk = perk
+                        Perk = perkLink.FormKey
```

Comment:
This line I'm not happy about.  I would've hoped `Perk = perkLink` would have worked, thanks to an implicit operator in FormLink's code.. but it doesn't seem to be happy about that.  This will have to do for now

Reply:
Eh, it looks like it's because C# doesn't like doing implicit operations on interfaces, which is silly imo.  I'll see if i can sidestep somehow, but probably not